app/channels/concurrent_editing_channel.rb

```diff
@@ -0,0 +1,13 @@
+class ConcurrentEditingChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "concurrent_editing_channel"
+  end
+
+  def unsubscribed
+    # Any cleanup needed when channel is unsubscribed
+  end
+
+  def sync changes
```

Comment:
Whenever changes are made, this method is called and it broadcasts the updated data to the channel.